device/mellanox/x86_64-nvidia_sn2201-r0/pmon_daemon_control.json

```diff
@@ -0,0 +1 @@
+../x86_64-mlnx_msn2700-r0/pmon_daemon_control.json
```

Comment:
please check path. might be that you need extra ../ as in other links

Reply:
I verified and the pah is valid. 
The red color disturbed me so I looked on other systems that have soft link and it is red there as well.
see for example: https://github.com/Azure/sonic-buildimage/blob/bf816964137444a1f1145656c2d4b60254620a8f/device/mellanox/x86_64-mlnx_msn3420-r0/thermal_policy.json